core/src/test/java/com/devonfw/tools/solicitor/componentinfo/scancode/ScancodeComponentInfoAdapterTest.java

```diff
@@ -53,7 +53,7 @@ public void setup() {
     this.fileScancodeRawComponentInfoProvider.setRepoBasePath("src/test/resources/scancodefileadapter/Source/repo");
 
     this.uncuratedScancodeComponentInfoProvider = new UncuratedScancodeComponentInfoProvider(
-        this.fileScancodeRawComponentInfoProvider, packageURLHandler);
+        this.fileScancodeRawComponentInfoProvider, packageURLHandler, null);
```

Comment:
This makes the test fail with NullPointerException. Please inject the "real thing" (probably needs  construction of objects to be slighty rearranged.)